esmvalcore/cmor/_fixes/cmip6/fgoals_f3_l.py

```diff
@@ -0,0 +1,41 @@
+"""Fixes for CMIP6 FGOALS-f3-L."""
+import cftime
+import numpy as np
+
+from ..fix import Fix
+
+
+class AllVars(Fix):
+    """Fixes for all vars."""
+    def fix_metadata(self, cubes):
+        """Fix parent time units.
+
+        FGOALS-f3-L Amon data may have a bad time bounds spanning 20 days.
+
+        Parameters
+        ----------
+        cubes : iris.cube.CubeList
+            Input cubes.
+
+        Returns
+        -------
+        iris.cube.CubeList
+        """
+        for cube in cubes:
+            if cube.attributes['table_id'] == 'Amon':
+                time = cube.coord('time')
+                if cube.attributes['table_id'] == 'Amon' and \
```

Comment:
you are checking for this already two lines up

Reply:
Thanks!